### PR TITLE
Update readme.html

### PR DESCRIPTION
**Pull request by ggarber**

### CODE REVIEW

File: docs/readme.html [hunk 1]
```diff
@@ -187,7 +187,7 @@
   and security.
 
   ```js
-  let config = {iceServers: [{url: 'stun:stun.l.google.com:19302'}]};
+  let config = {iceServers: [{urls: 'stun:stun.l.google.com:19302'}]};
```
Comment (ggarber) [inline docs/readme.html:190]: url is deprecated

File: docs/readme.html [hunk 2]
```diff
@@ -243,8 +245,6 @@
   pc.setLocalDescription(myOffer);
   ```
 
-  _setLocalDescription_ indirectly controls the candidate gathering process.
```
Comment (ggarber) [inline docs/readme.html:246]: AFAIK The gathering can start before setLocalDescription as an optimization to make the establishment faster:
https://tools.ietf.org/html/draft-ietf-rtcweb-jsep-19#section-3.5.4

**Comment by ggarber** [comment]
Apart from the changes i would say that:
- The trapezoid model (the first drawing) doesn't really exist in real products.  It is legacy apart from in telco/ SIP's mindset.
- For the interconnection with SIP (the second drawing) you don't really need a media relay unless you have legacy endpoints that cannot support things like DTLS or ICE that are required in WebRTC but are not specific of WebRRTC.

Have f1n!

File: docs/readme.html (hunk 2 above)
Comment (juandebravo) [reply docs/readme.html:246]: ok I'll test it again, in my tests it didn't happen before `setLocalDescription` but perhaps I was missing something. thanks!

**Review by juandebravo — APPROVED** [review]
* Trapezoid model: Agree, I referred to it as a model to expose that your web server can include multiple instances for scaling.

* Interconnection with SIP: agree, I'll include it in the slide but I referred in the talk to the points you mentioned (you only need relay if there's any incompatibility)

File: docs/readme.html (hunk 2 above)
Comment (juandebravo) [reply docs/readme.html:246]: @ggarber, good news for M-59 :)

https://www.chromestatus.com/feature/4973817285836800
https://bugs.chromium.org/p/chromium/issues/detail?id=673395
https://www.w3.org/TR/webrtc/#dom-rtcconfiguration-icecandidatepoolsize

**Pull request merged**